_drafts/SC3-3-2-form-field-has-name.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Form field has accessible name
+description: |
+  Each form field element has an accessible name
+
+success_criterion:
+- 3.3.2 # Labels or Instructions
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rules applies to:
```

Comment:
I don't think this list passes the "plain language" requirement.

Reply:
@WilcoFiers are you referring to the use of a ":" and a list? I was under the impression that plain language meant readable and understandable and I think that creating a single paragraph here goes against that from a readability perspective.